odoo/addons/base/models/ir_translation.py

```diff
@@ -923,3 +923,43 @@ def get_web_translations_hash(self, mods, lang):
             'multi_lang': len(self.env['res.lang'].sudo().get_installed()) > 1,
         }
         return hashlib.sha1(json.dumps(translation_cache, sort_keys=True).encode()).hexdigest()
+
+    @api.model
+    def _guess_language_of_terms(self, terms, types=None):
+        """ Guess the language of the terms based on the translations stored in the database.
```

Comment:
You should have a new line after the first line of the docstring (same for other docstring)
```
"""Guess the language of the terms based on the translations stored in the database.

Note that the search relies on some non indexed fields.
:param tuple(str) terms: terms used in the application for which we want to guess the language
:param set types: if set, limit the search to the given types (increase performance)
:return tuple(str, float):
    - best_candidate (str): the best language (code) candidate or None if none (ex.: fr_BE)
    - best_candidate_term_ratio (float): ratio of term found in the database for the best_candidate language
"""
```

Reply:
Thanks for the advice, noted and changed.